content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md

```diff
@@ -107,12 +118,18 @@ distribution may provide its own packages.
 
 ## Can I continue to use the legacy package repositories?
 
+_(updated on March 26, 2024)_
+
+**The legacy Google-hosted repositories went away on March 4, 2024. It's not possible
+to install Kubernetes packages from the legacy Google-hosted package repositories any
+longer.**
+
 ~~The existing packages in the legacy repositories will be available for the foreseeable
 future. However, the Kubernetes project can't provide _any_ guarantees on how long
 is that going to be. The deprecated legacy repositories, and their contents, might
 be removed at any time in the future and without a further notice period.~~
 
-**UPDATE**: The legacy packages are expected to go away by the end of February 2024.
+~~**UPDATE**: The legacy packages are expected to go away by the end of February 2024.~~
```

Comment:
(nit)
I'd remove this line outright

Reply:
https://github.com/kubernetes/website/pull/45705